modules/virt-installing-virt-operator.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * virt/install/installing-virt-web.adoc
+
+[id="virt-installing-virt-operator_{context}"]
+= Installing the {VirtProductName} Operator
+
+You can install the *{VirtProductName}* Operator from the {product-title} web console.
+
+.Procedure
+
+. Open a browser window and log in to the {product-title} web console.
+
+. From the *Administrator* perspective, click *Operators* -> *OperatorHub*.
+
+. In the *Filter by keyword* field, type *{VirtProductName}*.
+
+. Select the *{VirtProductName}* tile.
+
+. Read the information about the Operator and click *Install*.
+
+. On the *Install Operator* page:
+
+.. Select *stable* from the list of available *Update Channel* options. This ensures that you install the version of {VirtProductName} that is compatible with your {product-title} version.
+
+.. For *Installed Namespace*, ensure that the *Operator recommended namespace* option is selected. This installs the Operator in the mandatory `openshift-cnv` namespace, which is automatically created if it does not exist.
++
+[WARNING]
+====
+Attempting to install the {VirtProductName} Operator in a namespace other than
+`openshift-cnv` causes the installation to fail.
+====
+
+.. For *Approval Strategy*, ensure that *Automatic*, which is the default value, is selected. {VirtProductName} automatically updates when a new z-stream release is available.
```

Comment:
now that we have a single OLM channel named `stable`, this means that it will also automatically update to the next minor version after an OCP upgrade.